### PR TITLE
[snappass] Add SnapPass chart

### PR DESCRIPTION
**Pull request by anothertobi**
Signed-off-by: Tobias Nehrlich <tobias.nehrlich@vshn.ch>

#### What this PR does / why we need it:
Adds a Helm chart for https://github.com/pinterest/snappass

#### Checklist

- [x] [DCO](https://github.com/appuio/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR contains starts with chart name e.g. `[chart]`

### CODE REVIEW

File: snappass/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+                key: "redis-password"
+          {{- end }}
+          - name: SNAPPASS_REDIS_DB
+            value: "{{ $redis.db }}"
+          {{- if $redis.prefix }}
+          - name: REDIS_PREFIX
+            value: "{{ $redis.prefix }}"
+          {{- end }}
+          command: ["/bin/sh"]
+          args: ["-c", "REDIS_URL=redis://redis:$(REDIS_PASSWORD)@$(REDIS_HOST):$(REDIS_PORT)/$(SNAPPASS_REDIS_DB) snappass"]
```
Comment (srueg) [inline snappass/templates/deployment.yaml]: Could please test if signals (i.e. SIGTERM) are properly forwarded to the process?
If not we might use `exec snappass`

File: snappass/templates/deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+          ports:
+            - name: http
+              containerPort: 5000
+              protocol: TCP
+          livenessProbe:
```
Comment (srueg) [inline snappass/templates/deployment.yaml]: Please add a delay of at least 60s

File: snappass/values.yaml [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+  #  - secretName: chart-example-tls
+  #    hosts:
+  #      - chart-example.local
+
+resources:
+  requests:
+    cpu: 100m
+    memory: 128Mi
+  limits:
+    cpu: 100m
```
Comment (srueg) [inline snappass/values.yaml]: Maybe a little higher limits would be ok.

File: snappass/templates/deployment.yaml (hunk 1 above)
Comment (anothertobi) [reply snappass/templates/deployment.yaml]: SnapPass itself can't handle signals properly. Anyways, I added `exec env`.

**Pull request merged**